easybuild/easyblocks/b/bwise.py

```diff
@@ -0,0 +1,120 @@
+##
+# Copyright 2018 Free University of Brussels
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+@author: Ward Poelmans (Free University of Brussels)
+"""
+import glob
+import os
+import stat
+
+from easybuild.easyblocks.generic.makecp import MakeCp
+from easybuild.framework.easyconfig import CUSTOM
+from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.filetools import adjust_permissions, apply_regex_substitutions, change_dir
+from easybuild.tools.filetools import copy_file, which, write_file
+from easybuild.tools.modules import get_software_root
+
+
+class EB_BWISE(MakeCp):
+    """
+    Custom easyblock to install BWISE
+    """
+    @staticmethod
+    def extra_options():
+        """Change default values of options"""
+        extra = MakeCp.extra_options()
+        # files_to_copy is not mandatory here
+        extra['files_to_copy'][2] = CUSTOM
+        return extra
+
+    def build_step(self):
+        """Run multiple times for different sources"""
+
+        # BCALM is a git submodule of BWISE but we use it as a dependency
```

Comment:
Is it worth mentioning *why* BCALM is used as a dep, rather than just being included in the BWISE installation like you do with BGREAT & BTRIM?